data_updates/Python/iati_rhfp.py

```diff
@@ -0,0 +1,168 @@
+import os
+import requests
+import glob
+from requests.auth import HTTPBasicAuth
+import pandas as pd
+from github import Github, InputGitTreeElement
+from django.conf import settings
+
+
+donors_selected = pd.read_csv("https://staging-ddw.devinit.org/api/export/1232")
```

Comment:
Is this meant to be a staging URL or was that just for testing?

Reply:
Resolved below @edwinmp